deploy/charts/rook-ceph/values.yaml

```diff
@@ -77,9 +77,14 @@ containerSecurityContext:
 # -- If true, loop devices are allowed to be used for osds in test clusters
 allowLoopDevices: false
 
+# ceph-csi operator overrides
 ceph-csi-operator:
-  nameOverride: ceph-csi-operator
-  fullnameOverride: ceph-csi-operator
+  nameOverride: ceph-csi
```

Comment:
To have consistent names between manifest-based and helm-based installs, we need to use the `ceph-csi` prefix rather than `ceph-csi-operator`.

Reply:
with my latest push, it should be resolved